nobrainer/models/meshnetwn_old.py

```diff
@@ -0,0 +1,261 @@
+"""MeshNet implemented in TensorFlow.
```

Comment:
should this file be included in the repo? does `nobrainer/models/meshnetwn.py` supersede it?

Reply:
This file can be removed.